src/pages/components/content-block-simple.mdx

```diff
@@ -32,9 +33,22 @@ Content block &mdash; simple with aside elements is also used for introducing pa
 
 ![Image of content block simple with aside elements](../../images/pattern/content-block-simple/content-block-simple-with-aside-elements.jpg)
 
+## Content guidance
+
+| Field | Field type | Required   | Cardinality | Max character size (English / translated) |
+| -------- | --------- | ---------- | ---------- | ----------|
+| Headline | Short copy | Yes | 1 | XXS (40 / 55)|
+| TOC headline override | Short copy | No | 1 | Mini (25 / 35)|
```

Comment:
I saw this in another one. Is this a mistake copy over from something else?

Reply:
I just mentioned this in another PR